inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/model/AnnotationObject.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Objects;
 
 public class AnnotationObject
-    implements Serializable
+    implements Serializable, Comparable<AnnotationObject>
```

Comment:
I would not implement Comparable, but create a `Comparator `where you need it, as this `compareTo` does one of many things you might want to compare.

Reply:
Removed the Comparable interface from the class.